statsmodels/graphics/tests/test_tsaplots.py

```diff
@@ -51,8 +51,8 @@ def test_plot_month():
     dta = sm.datasets.elnino.load_pandas().data
     dta['YEAR'] = dta.YEAR.astype(int).apply(str)
     dta = dta.set_index('YEAR').T.unstack()
-    dates = lmap(lambda x : pd.datetools.parse('1 '+' '.join(x)),
```

Comment:
I suspect you are trying to use this.

```
In [3]: dateutil.parser.parse('1990')
Out[3]: datetime.datetime(1990, 10, 28, 0, 0)
```

which you should just use directly.